### ISSUE TITLE
Unbalanced Data - Change Loss Metric

### ISSUE THREAD

**Issue by chris-ml** [hyperopt/hyperopt-sklearn#46]
Hyperopt-sklearn for classification uses accuracy as the scorer. I'm using it on unbalanced data, so accuracy seems like a poor choice, and would prefer to use F1. Is it fine to  just switch `accuracy_score` for `f1_score` in estimator.py line 288. Or is there a better approach, am I missing something?

**Comment by bjkomer** [comment]
I was working on a way to supply custom loss functions a [while ago](https://github.com/bjkomer/hyperopt-sklearn/commit/879ed4cee0576d2f85f74cda2eda1d9efc56b412), but it looks like I never got around to merging it into master. I'll get that in soon, but in the meantime just changing that line should work fine.

**Comment by chris-ml** [comment]
Cool, thanks for speedy reply ;)
